import_obfuscation.go

```diff
@@ -44,34 +47,62 @@ type privateImports struct {
 	privateNames []string
 }
 
-func obfuscateImports(objPath, importCfgPath string) (map[string]string, error) {
+func appendPrivateNameMap(nameMap map[string]string, packageDirectory string) error {
+	file, err := os.Open(filepath.Join(packageDirectory, garbleMapFile))
+	if errors.Is(err, os.ErrNotExist) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
+
+	var localMap map[string]string
+	if err = gob.NewDecoder(file).Decode(&localMap); err != nil {
```

Comment:
`:=`, not `=`, here and in all other places. otherwise you are modifying the parent error variable.